config.ini

```diff
@@ -116,6 +116,20 @@ pause_threshold = 0.5
 ;   Recommended: 30
 listen_timeout = 30
 
+; whisper_type
+;   Advanced users only.  Allows using whispercpp (https://github.com/ggerganov/whisper.cpp) in server mode instead of default faster_whisper.
+;   The main benefits would be to reduce vram usage when using larger whisper models, to enable use of distil-whisper models,
+;   To share a whisper speech to text service between AI mods like mantella and herika, or run the whispercpp server in a cloud service.
+;   In whispercpp server mode, the server settings, not the ones above, will control the model you use and cpu vs. gpu usage.  
+;   You are expected to "bring your own server" and have it running while running Mantella.
```

Comment:
maybe change the word 'it' to 'whispercpp' or say 'bring your own whispercpp server' to make it perfectly clear

Reply:
Good idea, done thanks!